docs/cli/usage/containers.md

```diff
@@ -147,7 +147,7 @@ This `docker-compose.yml` file would be placed in your `./amplify/backend/api/<n
    Dockerfile
    package.json
    index.js
-/python
+ /python
```

Comment:
is the whitespace intentional here?

Reply:
Yes, the live version is _missing_ to indicate the `/python` dir at the same level as the `/express` dir.

ref: https://docs.amplify.aws/cli/usage/containers#deploy-multiple-containers